Dockerfile

```diff
@@ -3,7 +3,10 @@ FROM php:7.1-fpm-alpine
 RUN apk add --no-cache --virtual .persistent-deps \
 		git \
 		icu-libs \
-		zlib
+		zlib \
+		wget \
+		ca-certificates \
```

Comment:
@danjesus I added this on commit ac5cc26 
Can you please test PR #52? 
Thank you :)